src/topology/homotopy/induced_maps.lean

```diff
@@ -0,0 +1,190 @@
+/-
+Copyright (c) 2022 Praneeth Kolichala. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Praneeth Kolichala
+-/
+import topology.homotopy.product
+import topology.homotopy.equiv
+import topology.homotopy.casts
+import category_theory.equivalence
+
+/-!
+# Homotopic maps induce naturally isomorphic functors
+
+## Main definitions
+
+  - `fundamental_groupoid_functor.homotopic_maps_nat_iso H` The natural isomorphism
+    between the induced functors `f : π(X) ⥤ π(Y)` and `g : π(X) ⥤ π(Y)`, given a homotopy
+    `H : f ∼ g`
+
+  - `fundamental_groupoid_functor.equiv_of_homotopy_equiv hequiv` The equivalence of the categories
+    `π(X)` and `π(Y)` given a homotopy equivalence `hequiv : X ≃ₕ Y` between them.
+
+## Implementation notes
+  - In order to be more universe polymorphic, we define `ulift_homotopy`
+  which lifts a homotopy from `I × X → Y` to `(Top.of ((ulift I) × X)) → Y`
+-/
+
+noncomputable theory
+
+universe u
+
+namespace unit_interval
+open_locale unit_interval
+
+/-- The path 0 ⟶ 1 in I -/
+def path_zero_one : path (0 : I) 1
+:= { to_fun := id, source' := rfl, target' := rfl }
```

Comment:
```suggestion
def path_zero_one : path (0 : I) 1 := { to_fun := id, source' := rfl, target' := rfl }
```

Reply:
I suppose that this path will be used a lot in the future. Should it be called `path_0_1` or even `path01` for sake of brevity?